feel/feelcore/environment.hpp

```diff
@@ -611,7 +611,11 @@ class FEELPP_EXPORT Environment : boost::noncopyable
     //! the application directory, application results are stored there
     //! the directory is controlled by changeRepository
     static std::string appRepository();
-    
+
+    //! the application directory without np, application results are stored there
+    //! the directory is controlled by changeRepository
+    static std::string appRepositoryWithoutNumProc();
```

Comment:
why this change? what is the rationale?
is it to reduce the number of directories?
we have too many of them anyway. We need to radical change in the data handling

Reply:
it's for store expression (or other data) which not depends on number of proc (it was implemented in toolboxes core but it easier to manage this in the Environment.